flink-clients/src/test/java/org/apache/flink/client/program/DefaultPackagedProgramRetrieverITCase.java

```diff
@@ -394,17 +393,19 @@ void testWithJobClassAndMultipleEntryClassesOnSystemClasspath()
     }
 
     @Test
-    void testRetrieveCorrectUserClasspathsWithoutSpecifiedEntryClass()
-            throws IOException, FlinkException, ProgramInvocationException {
+    void testRetrieveCorrectUserClasspathsWithoutSpecifiedEntryClass() throws Exception {
         final PackagedProgramRetriever retrieverUnderTest =
                 DefaultPackagedProgramRetriever.create(
                         singleEntryClassClasspathProvider.getDirectory(),
                         null,
                         ClasspathProviderExtension.parametersForTestJob("suffix"),
                         new Configuration());
-        final JobGraph jobGraph = retrieveJobGraph(retrieverUnderTest, new Configuration());
+        final StreamGraph streamGraph =
+                retrieveStreamGraph(retrieverUnderTest, new Configuration());
         final List<String> actualClasspath =
-                jobGraph.getClasspaths().stream().map(URL::toString).collect(Collectors.toList());
+                streamGraph.getClasspaths().stream()
+                        .map(URL::toString)
+                        .collect(Collectors.toList());
```

Comment:
can we extract this
```java
streamGraph.getClasspaths().stream()
                        .map(URL::toString)
                        .collect(Collectors.toList());
```
into a separate method, it is duplicated in this class multiple times

Reply:
Done.